File: src/CoinyProject.Client/src/app/album/albums-view/album-view-get.model.ts
import {UsernameModel} from "../../services/username.model";

export interface AlbumViewGetDto {
  id: string;
  name: string;
  description: string;
  rate: number;
  status: string;
  author: UsernameModel;
  updatedAt: string;
  imagesUrls: string[];
  currentImageIndex: number;
}

File: src/CoinyProject.Application/Features/Albums/Models/AlbumViewGetModel.cs
using CoinyProject.Application.Dto.User;
using CoinyProject.Domain.Enums;

namespace CoinyProject.Application.Features.Albums.Models
{
    public class AlbumViewGetModel
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public int Rate { get; set; }
        public AlbumStatus Status { get; set; }
        public UserNameGetDto Author { get; set; }
        public DateTime UpdatedAt { get; set; }
        public IEnumerable<string> ImagesUrls { get; set; } = new List<string>();
    }
}